src/PermissionRegistrar.php

```diff
@@ -25,7 +25,7 @@ class PermissionRegistrar
     /** @var string */
     protected $roleClass;
 
-    /** @var Collection|null */
+    /** @var array|null */
```

Comment:
Um ... but it's actually still a collection internally ...

Reply:
You are right, I see that the problem starts with the new version of phpstan

Reply:
Fixed